tests/conflux/config.py

```diff
@@ -27,8 +27,6 @@
 
 small_local_test_conf = dict(
```

Comment:
Is reorg_test considered "small_local_test" like the rest 4 tests? Maybe you want to create an accurate category?